### PR TITLE
handle internal and external links on markdown content

### PR DESCRIPTION
**Pull request by pnavarrc**
The markdown component has a custom renderer for links. This PR differentiates internal and external links by parsing the `href` parameter as a [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) and checking if the link includes a hostname.

Internal links use the `HashLink` component, which scrolls to the specific link (let's say `/glossary#superspreader`, external links will open on a new tab (and have `rel="noopened noreferrer"`)

### CODE REVIEW

**Review by chasulin — COMMENTED** [review]
looks good so far!! 2 things:
-  the offset isnt exactly right. when clicking an internal link to another glossary term, it doesnt exactly scroll to the right place
- we can add this to the list, but it might be useful to use the [accordion's extended prop](https://material-ui.com/api/accordion/) to open the term that is clicked on/scrolled to.

File: src/components/SharedComponents/Accordion.tsx [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+import { AccordionDetails, AccordionProps } from '@material-ui/core';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import {
   StyledMuiAccordion,
   StyledAccordionSummary,
   MarkdownBody,
 } from './Accordion.style';
 
-const StyledAccordion = (props: {
+export interface StyledAccordionProps extends Omit<AccordionProps, 'children'> {
```
Comment (chasulin) [inline src/components/SharedComponents/Accordion.tsx:10]: does this update syntax serve a specific purpose in this case?

File: src/components/SharedComponents/Accordion.tsx [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+}
 
+const StyledAccordion: React.FC<StyledAccordionProps> = ({
+  summaryText,
+  detailText,
+  ...otherProps
+}) => {
   return (
-    <StyledMuiAccordion>
+    <StyledMuiAccordion {...otherProps}>
```
Comment (chasulin) [inline src/components/SharedComponents/Accordion.tsx:21]: same as above q ^ :
does the updated syntax + addition of `{...otherProps}` serve a specific purpose here?

File: src/components/SharedComponents/Accordion.tsx (hunk 1 above)
Comment (pnavarrc) [reply src/components/SharedComponents/Accordion.tsx:10]: yes! It allows us to pass the `expanded` and `onChange` props to the `StyledAccordion`, as well as any other `MuiAccordion` props

**Review by chasulin — APPROVED** [review]

**Pull request merged**